core/src/main/java/eu/fasten/core/data/metadatadb/MetadataDao.java

```diff
@@ -883,18 +886,16 @@ public String getPackageVersions(String packageName, int offset, int limit) {
         Packages p = Packages.PACKAGES;
         PackageVersions pv = PackageVersions.PACKAGE_VERSIONS;
         ArtifactRepositories ar = ArtifactRepositories.ARTIFACT_REPOSITORIES;
-
         // Query
         var queryResult = context
                 .select(pv.ID, pv.PACKAGE_ID, pv.CG_GENERATOR, pv.VERSION, ar.REPOSITORY_BASE_URL, pv.ARCHITECTURE, pv.CREATED_AT)
                 .from(p)
                 .innerJoin(pv).on(p.ID.eq(pv.PACKAGE_ID))
-                .innerJoin(ar).on(pv.ARTIFACT_REPOSITORY_ID.eq(ar.ID))
```

Comment:
Why `innerJoin` is changed to `leftJoin` here?

Reply:
Well, this method is used to retrieve all package versions of a specific package name. With inner join, it would return only package versions which have an artifact repository, which is a java  language specific attribute.  Therefore, with inner join, for C and Python the querry would not return any package version. To tackle this issue, changed the inner join to left join, so that to return also package versions that do not include an artifact repository attribute & table population.